lib/screenplay_web/auth_manager/auth_manager.ex

```diff
@@ -12,6 +12,15 @@ defmodule ScreenplayWeb.AuthManager do
     "pa-message-admin" => :pa_message_admin
   }
 
+  def max_session_time do
+    Application.get_env(:screenplay, __MODULE__)[:max_session_time]
+  end
+
+  def idle_time do
+    Application.get_env(:screenplay, __MODULE__)[:idle_time]
+  end
```

Comment:
Noting this just in case. A straight copy/paste of these `Application` calls is not quite what is needed. Since the `AuthManager` config is split between compile time and runtime, we needed to use a path in the `compile_env` calls.